pkg/operator/ceph/object/notification/controller.go

```diff
@@ -197,8 +203,10 @@ func (r *ReconcileNotifications) reconcile(request reconcile.Request) (reconcile
 		if err != nil {
 			return reconcile.Result{}, errors.Wrapf(err, "failed to provision CephBucketNotification %q for ObjectBucketClaims %q", bnName, bucketName)
 		}
+		r.recorder.Eventf(notification, "Normal", "Reconcile", "Notification %q provisioned on bucket %q", bnName, bucketName)
```

Comment:
Perhaps we might consider a `"Provisioned"` event type here instead of reusing `"Reconcile"`. Also should probably make this part of the API here: https://github.com/BlaineEXE/rook/blob/3f766406cb4a0e780a3a02e0e03a5ebb277ed08a/pkg/apis/ceph.rook.io/v1/types.go#L401-L426


### Here are some events for comparison...

Pod:
```
Events:
  Type    Reason     Age   From               Message
  ----    ------     ----  ----               -------
  Normal  Scheduled  24s   default-scheduler  Successfully assigned rook-ceph/rook-ceph-rgw-my-store-a-85cf66b968-44hqh to minikube
  Normal  Pulled     24s   kubelet            Container image "quay.io/ceph/ceph:v16.2.6" already present on machine
  Normal  Created    24s   kubelet            Created container chown-container-data-dir
  Normal  Started    24s   kubelet            Started container chown-container-data-dir
  Normal  Pulled     23s   kubelet            Container image "quay.io/ceph/ceph:v16.2.6" already present on machine
  Normal  Created    23s   kubelet            Created container rgw
  Normal  Started    23s   kubelet            Started container rgw
```
  
  replication controller :
  ```
  Events:
  Type    Reason            Age   From                   Message
  ----    ------            ----  ----                   -------
  Normal  SuccessfulCreate  11m   replicaset-controller  Created pod: rook-ceph-rgw-my-store-a-85cf66b968-44hqh
```
  
  Job:
  ```
  Events:
  Type    Reason            Age   From            Message
  ----    ------            ----  ----            -------
  Normal  SuccessfulCreate  22m   job-controller  Created pod: rook-ceph-osd-prepare-minikube--1-s57vq
  Normal  Completed         22m   job-controller  Job completed
```